search/geocoder.cpp

```diff
@@ -6,6 +6,7 @@
 #include "search/features_layer_matcher.hpp"
 #include "search/house_numbers_matcher.hpp"
 #include "search/locality_scorer.hpp"
+#include "search/pre_ranker.hpp"
```

Comment:
Well we should include it somewhere, shouldn't we? Note that it is not included in geocoder.hpp